Packs/GSuiteAdmin/Integrations/GSuiteAdmin/README.md

```diff
@@ -1870,4 +1870,619 @@ There is no context output for this command.
+
+#### Base Command
+
+`gsuite-chromebrowserdevice-move-ou`
+
+##### Required Permissions
+`https://www.googleapis.com/auth/admin.directory.device.chromebrowsers`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| customer_id | The customerId is a unique identifier of the customer's Google account. | Required | 
+| resource_ids | List of unique device IDs (comma separated) of Chrome Browser Devices to move. A maximum of 600 browsers may be moved per request. | Required | 
+| org_unit_path | Destination organization unit to move devices to. Full path of the organizational unit or its ID prefixed with 'id:'. | Required | 
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+```!gsuite-chromebrowserdevice-move-ou customer_id="test" resource_ids="11111" org_unit_path="/testing"```
+
+#### Human Readable Output
+> Chrome browser devices have been moved to the new organization unit ORGANIZATIONAL_UNIT_PATH.
+
+### gsuite-chromebrowserdevice-list
+
+***
+Retrieve all Chrome browser devices for an account or a specific Chrome browser device
+
+#### Base Command
+
+`gsuite-chromebrowserdevice-list`
+
+##### Required Permissions
+`https://www.googleapis.com/auth/admin.directory.device.chromebrowsers`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| customer_id | The customerId is a unique identifier of the customer's Google account. | Required | 
+| device_id | The unique ID of the device. | Optional | 
+| order_by | Field according to which to order by. Possible values are: id, last_sync, machine_name, extension_count, policy_count, os_version, last_signed_in_user, annotated_user, annotated_location, annotated_asset_id, notes, browser_version_channel, org_unit, enrollment_date, save_browsing_clickthrough, platform_major_version, last_activity, browser_version_sortable, os_version_sortable. | Optional | 
+| org_unit_path | The full path of the organizational unit or its unique ID. | Optional | 
+| projection | Restrict information returned to a set of selected fields. "BASIC" - Includes only the basic metadata fields (i.e., the ones found in the columns of the admin console browser list as enumerated above). "FULL" - Includes all metadata fields (as enumerated above). Possible values are: BASIC, FULL. | Optional | 
+| query | Search string using the list page query language described in the section below (Filter Query Language). | Optional | 
+| sort_order | Whether to return results in ascending or descending order. Must be used with the orderBy parameter. "ASCENDING" - Ascending order. "DESCENDING" - Descending order. Possible values are: ASCENDING, DESCENDING. | Optional | 
+| page_size | The maximum number of policy schemas to return, default and maximum are 100. Default is 100. | Optional | 
+| page_token | Page Token. | Optional | 
+| limit | Limit. | Optional | 
```

Comment:
Limit of what? The number of devices per retrieval? The number of devices per page?